test/SourceKit/NameTranslation/basic.swift

```diff
@@ -66,6 +67,7 @@ class MyDerived: FooClassDerived {
 // CHECK9: fooProperty11
 // CHECK10: init(nstanceFunc21:)
 // CHECK11: init(lassDerived2:)
+// CHECK11-1: FooClassDerived2
```

Comment:
You are right. We need to add the similar logic to handle swift->objc translation as well.